docs/release-notes/december-2023.md

```diff
@@ -0,0 +1,136 @@
+---
+slug: december-2023
+append_help_link: true
+hide_title: true
+description: >-
+    Release notes include the changes, fixes, and additions in specific versions of Semgrep.
+toc_max_heading_level: 3
+title: December 2023
+tags:
+  - Release notes
+---
+
+# Semgrep release notes for December 2023
+
+## 🔧 OSS Engine
+
+* The following versions of the OSS Engine were released in December 2023:
+
+  * [<i class="fas fa-external-link fa-xs"></i>
+    1.52.0](https://github.com/returntocorp/semgrep/releases/tag/v1.52.0)
+  * [<i class="fas fa-external-link fa-xs"></i>
+    1.53.0](https://github.com/returntocorp/semgrep/releases/tag/v1.53.0)
+  * [<i class="fas fa-external-link fa-xs"></i>
+    1.54.0](https://github.com/returntocorp/semgrep/releases/tag/v1.54.0)
+  * [<i class="fas fa-external-link fa-xs"></i>
+    1.54.1](https://github.com/returntocorp/semgrep/releases/tag/v1.54.1)
+
+## 🌐 Cloud Platform
+
+### Added
+
+* Semgrep IDE integrations now cache information about the current repository so that it
+  does not traverse the entire repository on every scan to determine if the
+  files are valid targets for scanning; this improves scan times.
+* Users can now ignore findings locally in Semgrep IDE extensions. These changes
+  persist between restarts.
+* The metrics collected now include more granular information to help
+differentiate scans using different engine capabilities, such as intraprocedural
+scans without secrets validation versus intraprocedural scans *with* secrets
+validation.
+* **CLI tool**: Added new `semgrep test` subcommand, which is an alias for
+`semgrep scan --test`. **Note**: When running Semgrep on a test directory, use
```

Comment:
```suggestion
`semgrep scan --test`. **Note**: If the **name** of the directory you are scanning is `test`, use `semgrep scan test` to avoid confusion with the new `semgrep test` subcommand.
```